front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -59,7 +59,6 @@ const TimesStops = <T extends TimesStopsRow>({
       }}
       stickyRightColumn={stickyRightColumn}
       lockRows
-      height={600}
```

Comment:
It seems like without this, `height` defaults to 400px, which isn't enough for browser windows with a small height as you noted in the PR description.

Also, this reduces the height in the times table in the simulation results view, so reading the table there is less comfortable.

I think ideally we'd keep the height to 600px for the simulation results table, and use CSS to make the table use the full available height (but not more!) in the train edition modal.